arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -220,12 +219,6 @@
         "responses": {
           "202": {
             "description": "Workspace collection deleted successfully"
-          },
-          "default": {
```

Comment:
why default response is removed, this will be used when there is error and, listed response is not returned.